src/Ssh/Ssh.AlcWrapper/SshServices.cs

```diff
@@ -0,0 +1,313 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.HybridCompute;
+using Microsoft.Azure.Management.HybridCompute.Models;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Exceptions;
+using System.Linq;
+using Microsoft.Rest.Azure;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Commands.Ssh
+{
+    //As Compute module must not reference to storage SDK, the types of parameters and return values of
+    //public APIs within this project should not expose any types defined in storage SDK.
+
+	public class ComputeClient
```

Comment:
Usually, editors are set to convert tab characters into spaces.  Not sure if that is true for Az cmdlets.
```suggestion
public class ComputeClient
```